packages-common/pluggable-widgets-tools/tests/commands.spec.js

```diff
@@ -33,9 +37,24 @@ describe("pluggable-widgets-tools commands", () => {
         rm(toolsPackagePath);
     });
 
-    describe.each(CONFIGS)("for %s %s widget created with generator %s", (platform, lang, version) => {
+    describe.each(CONFIGS)("for %s %s widget created with generator %s", (platform, boilerplate, lang, version) => {
         const widgetName = `generated_${version.replace(".", "_")}_${lang}_${platform}`;
         const workDir = join(tempdir(), `pwt_test_${Math.round(Math.random() * 10000)}`);
+        const promptAnswers = {
```

Comment:
- i think some of these answers are default => there is no need to specify them
- it is a question of taste, but since these answers make sense only inside `if` on line 63, i would rather put them there. To keep code more cohesive. But this is really possible only if we get rid of all defaults and make this list small again.
- ah you use them even outside of that `if`... But maybe will be easier to inline correct values to that place where you use them. Not sure.